R/principalcomponentanalysis.R

```diff
@@ -227,14 +242,14 @@ PrincipalComponentAnalysis <- function(jaspResults, dataset, options, ...) {
 
   coltitle <- if (options[["rotationMethod"]] == "orthogonal") "PC" else "RC"
   for (i in seq_len(ncol(loads)))
-    loatab$addColumnInfo(name = paste0("c", i), title = paste0(coltitle, i), type = "number", format = "dp:3")
+    loadTab$addColumnInfo(name = paste0("c", i), title = paste0(coltitle, i), type = "number", format = "dp:3")
```

Comment:
what's the reason for the manual formatting here?

Reply:
I changed the object names to camelCase. but I have done so anyways in the option renaming PR, that I am working on. So this will be "loadingsTable"... just for better readability.

Reply:
sorry, I did not mean that, the camelCase change is ok. I was just asking why do we need `format = "dp:3"` here

Reply:
I have no idea. I did not write most of this analysis :D

Reply:
ok, then just leave it as is